src/appengine/libs/gcs.py

```diff
@@ -35,6 +36,26 @@
     ['url', 'bucket', 'key', 'google_access_id', 'policy', 'signature'])
 
 
+def service_account_email():
```

Comment:
This could be moved to utils as a helper.

Reply:
Done.